src/manual/tasks/artifacts.md

```diff
@@ -0,0 +1,45 @@
+---
+filename: tasks/artifacts.md
+title: Artifacts
+order: 35
+---
+
+But by far the most commonly-used data from a run are its artifacts. These are
+HTTP entities containing output data from the task execution.
+
+Unlike most API methods which return a JSON body, requesting an artifact from
+the Queue service returns the artifact itself, possibly via one or more HTTP
+redirects. This means that -- at least for public artifacts which require no
+authentication -- any sufficiently robust HTTP client can download an artifact
+directly.
+
+**NOTE**: Not all clients are "sufficiently robust"! The artifact interface
+makes heavy use of redirects, and artifacts may make use of other web-standard
+features such as content encoding.  Like any distributed system, requests may
+fail, too, and a robust client should retry. Out of the box, `curl` and `wget`
+do not handle most of these cases.
+
+Taskcluster's Queue service supports a number of artifact types, including
+several cloud data-storage back-ends as well as two special types: errors and
+references. Error artifacts will always return an HTTP 403 (Forbidden), with
+message and details supplied by the task. Reference artifacts return a 303 (See
+Other) redirecting the client to another URL.
+
+## Public Artifacts
+
+While it is possible to create artifacts which require authorization to
+download, most artifacts are public. These are easily identified by the prefix
+`public/` in the artifact name. All other artifacts are private, and
+authorization will be required to read them.
```

Comment:
I think it's a little early in the manual to get into specific scopes.  I'll add a bit about how the authorization is by name, not by task.